pareto/utilities/earthquake_distance.py

```diff
@@ -0,0 +1,179 @@
+        "https://maps.texnet.beg.utexas.edu/arcgis/rest/services/catalog/catalog_all/MapServer/0/query"
+        + "?f=geojson"
+        "&where=Magnitude>={min_magnitude}"
+        + "&text="
+        + "&objectIds="
+        + "&time="
+        + "&timeRelation=esriTimeRelationOverlaps"
+        + "&geometry={lon},{lat}"
+        + "&geometryType=esriGeometryEnvelope"
+        + "&inSR="
+        + "&spatialRel=esriSpatialRelWithin"
+        + "&distance={max_radius_mi}"
+        + "&units=esriSRUnit_StatuteMile"
+        + "&relationParam="
+        + "&outFields=*"
+        + "&returnGeometry=true"
+        + "&returnTrueCurves=false"
+        + "&maxAllowableOffset="
+        + "&geometryPrecision="
+        + "&outSR="
+        + "&havingClause="
+        + "&returnIdsOnly=false"
+        + "&returnCountOnly=false"
+        + "&orderByFields="
+        + "&groupByFieldsForStatistics="
+        + "&outStatistics="
+        + "&returnZ=false"
+        + "&returnM=false"
+        + "&gdbVersion="
+        + "&historicMoment="
+        + "&returnDistinctValues=false"
+        + "&resultOffset="
+        + "&resultRecordCount="
+        + "&returnExtentOnly=false"
+        + "&sqlFormat=none"
+        + "&datumTransformation="
+        + "&parameterValues="
+        + "&rangeValues="
+        + "&quantizationParameters="
+        + "&featureEncoding=esriDefault"
+        + ""
+    )
+
+    earthquake_distances = []
+
+    for swd_latlon in swd_latlons:
+        swd_id = swd_latlon["id"]
+        swd_lat = swd_latlon["lat"]
+        swd_lon = swd_latlon["lon"]
+        url = texnet_api_url.format(
+            lat=swd_lat,
+            lon=swd_lon,
+            max_radius_mi=max_radius_mi,
+            min_magnitude=min_magnitude,
+        )
+
+        try:
+            request = urllib.request.Request(url)
+            with urllib.request.urlopen(request) as f:
+                response = json.load(f)
+        except:
+            raise
+        for feat in response["features"]:
+            eq_id = feat["id"]
+            props = feat["properties"]
+            coords = feat["geometry"]["coordinates"]
+
+            lat = coords[1]
+            lon = coords[0]
+            mag = props["Magnitude"]
+            time = datetime.datetime.fromtimestamp(props["Event_Date"] / 1000).strftime(
+                "%Y-%m-%d %H:%M:%S"
+            )
+            dist_mi = geod.line_length([swd_lon, lon], [swd_lat, lat]) * mi_per_m
+            earthquake_distances.append(
+                {
+                    "swd_id": swd_id,
+                    "eq_id": eq_id,
+                    "time": time,
+                    "distance_mi": dist_mi,
+                    "magnitude": mag,
+                }
+            )
+
+    return earthquake_distances
+
+
+# Example rune
+# swd_latlons = [
+#    {"id": 1, "lat": 34, "lon": -106},
+#    {"id": 2, "lat": 35, "lon": -105},
+# ]
+# earthquake_distances = calculate_earthquake_distances_texnet(swd_latlons)
```

Comment:
I tried running this example with `calculate_earthquake_distances_usgs` instead of `calculate_earthquake_distances_texnet` and it returned an empty list. Can you provide an example of using `calculate_earthquake_distances_usgs` that does not return an empty list, so we can be sure that it is working as intended?